dags/taar_weekly.py

```diff
@@ -0,0 +1,59 @@
+"""
+This configures a weekly DAG to run the TAAR Ensemble job off.
+"""
+from airflow import DAG
+from airflow.operators.sensors import ExternalTaskSensor
+from datetime import datetime, timedelta
+from airflow.operators.moz_databricks import MozDatabricksSubmitRunOperator
+from utils.mozetl import mozetl_envvar
+
+default_args_weekly = {
+    "owner": "frank@mozilla.com",
+    "depends_on_past": False,
+    "start_date": datetime(2019, 5, 31),
+    "email": ["telemetry-alerts@mozilla.com", "frank@mozilla.com"],
+    "email_on_failure": True,
+    "email_on_retry": True,
+    "retries": 2,
+    "retry_delay": timedelta(minutes=30),
+}
+
+
+
+taar_weekly = DAG(
+    "taar_weekly", default_args=default_args_weekly, schedule_interval="@weekly"
+)
+
+wait_for_clients_daily = ExternalTaskSensor(
+    task_id='clients_daily',
+    external_dag_id='main_summary',
+    external_task_id='clients_daily',
+    execution_delta=timedelta(days=-7, hours=-1), # main_summary waits one hour, execution date is beginning of the week
+    dag=taar_weekly)
+
+
+taar_ensemble = MozDatabricksSubmitRunOperator(
+    task_id="taar_ensemble",
+    job_name="TAAR Ensemble Model",
+    owner="vng@mozilla.com",
+    email=["vng@mozilla.com", "mlopatka@mozilla.com"],
+    execution_timeout=timedelta(hours=11),
+    instance_count=5,
+    instance_type="i3.2xlarge",
+    spot_bid_price_percent=100,
+    max_instance_count=60,
+    enable_autoscale=True,
+    start_date='20190527',
```

Comment:
What's this keyword for?

Reply:
https://github.com/apache/airflow/blob/master/airflow/models/baseoperator.py#L88

``` The ``start_date`` for the task, determines
        the ``execution_date`` for the first task instance.```